src/igvfd/audit/measurement_set.py

```diff
@@ -111,6 +111,12 @@ def audit_inconsistent_readout(value, system):
                 f'not expected to specify a data readout.'
             )
             yield AuditFailure('inconsistent readout', detail, level='NOT_COMPLIANT')
+    if assay_term == value.get('readout'):
```

Comment:
I think this condition belongs before line 107 - you are looking on an assay with readout, and it is one of the assays that are expected to have readout - so you check they are not identical.

Reply:
Fixed